### ISSUE TITLE
Game crashed after open overview of attractions.

### ISSUE THREAD

**Issue by jkroepke** [OpenRCT2/OpenRCT2#4603]
**OS:** Windows 10
**Version:** 0.0.5
**Commit/Build:** b15f8d0

Game crashed after open overview of attractions.

```
---------------------------
OpenRCT2
---------------------------
A crash has occurred and a dump was created at
....\OpenRCT2\9b0dfc58-959c-4443-a88d-fefcb284a8af(b15f8d0).dmp.

Please file an issue with OpenRCT2 on GitHub, and provide the dump and saved game there.

Version: 0.0.5
Commit: b15f8d0
---------------------------
OK   
---------------------------
```

- [ ] Reproducible in RCT2 (vanilla)?
- [X] Multiplayer?

**Steps to reproduce:**
1. Load savegame
2. Click "View all attractions" button on top left side

**Dump file / Save game:**

[OpenRCT2.zip](https://github.com/OpenRCT2/OpenRCT2/files/522927/OpenRCT2.zip)

**Comment by manuelVo** [comment]
Seems like this was fixed by b007aa9 (#4594).

**Comment by janisozaur** [comment]
Ah, yes, I didn't notice it was a build from before #4594.
